tests/integration/targets/homebrew_cask/defaults/main.yml

```diff
@@ -0,0 +1 @@
+cask: bitcoin-core
```

Comment:
Please pick something else that's less morally/ethically loaded. Thanks.

Reply:
done

Reply:
Thanks!